cypress/integration/settings.ts

```diff
@@ -410,7 +392,7 @@ context('Settings tests', () => {
 
       cy.get('.MuiTable-root tbody tr')
         .first()
-        .then(element => expect(element.text()).to.contain('submitted'));
+        .then((element) => expect(element.text()).to.contain('submitted'));
```

Comment:
Why do we need parenthesis here? Are they added by linting auto fix?

Reply:
Yes this is a change in the prettier that was introduced after we updated it. https://prettier.io/docs/en/options.html#arrow-function-parentheses